audioswitch/src/main/java/com/twilio/audioswitch/bluetooth/BluetoothHeadsetReceiver.kt

```diff
@@ -18,13 +18,17 @@ import android.media.AudioManager.SCO_AUDIO_STATE_ERROR
 import com.twilio.audioswitch.android.BluetoothDeviceWrapper
 import com.twilio.audioswitch.android.BluetoothIntentProcessor
 import com.twilio.audioswitch.android.LogWrapper
+import com.twilio.audioswitch.selection.AudioDeviceManager
 
 private const val TAG = "BluetoothDeviceReceiver"
 
 internal class BluetoothHeadsetReceiver(
     private val context: Context,
     private val logger: LogWrapper,
     private val bluetoothIntentProcessor: BluetoothIntentProcessor,
+    audioDeviceManager: AudioDeviceManager,
```

Comment:
Should this be private? If visibility is relaxed for testing then add `VisibleForTesting` annotation

Reply:
Ah yeah so this is actually not a property. It's just a constructor parameter so no visibility modifier is needed.